### ISSUE TITLE
Implement learning rate scheduler

### ISSUE THREAD

**Issue by adnan-umich** [adnan-umich/monai-train#19]
https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate

**Comment by adnan-umich** [comment]
my_optimizer = torch.optim.Adam( my_model.parameters(), 
                                 lr = 0.001, 
                                 weight_decay = 0.002)

my_lr_scheduler = torch.optim.lr_scheduler.StepLR( my_optimizer, 
                                                step_size = 50, 
                                                gamma = 0.1)

# train
...
my_optimizer.step()
my_lr_scheduler.step()

# get learning rate
my_lr = my_lr_scheduler.get_lr()
# or
my_lr = my_lr_scheduler.optimizer.param_groups[0]['lr']
